psd-web/app/assets/stylesheets/case-bar.scss

```
@@ -0,0 +1,10 @@
+.app--case-bar .app--case-bar-details {
+  display: block;
+}
+
+.app--case-bar .app--case-bar-title {
+  display: block;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
```

I think this should be:

```suggestion
.app-case-bar .app-case-bar__details {
  display: block;
}

.app-case-bar .app-case-bar__title {
  display: block;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}
```

...following the [BEM conventions](http://getbem.com/naming/). ie `app-` is the prefix, `__` is used to denote a child element, and `--` is used for a modifier.

---

👍 thanks for pointing to the docs.